tox.ini

```diff
@@ -12,6 +12,7 @@ deps=
     -r{toxinidir}/requirements_dev.txt
 
 commands =
+    inv cog -c true
```

Comment:
I'm a bit confused. On `tasks.py` , the cog task doesn't seem to have any parameter (other than the context):

```
@invoke.task
def cog(ctx):
    """ Executes cog on alfasim_sdk/alfacase/schema.py to generate the schema for strictyaml. """
    ctx.run(command=f"cog -rc {schema_file_path()}", warn=True)
```

What does `-c true` means?
Nevertheless, although those aliases are useful for daily usage, I suggest changing this with the full name instead of `-c`, so that the reader won't have to check

Reply:
Good catch, https://github.com/ESSS/alfasim-sdk/pull/108/commits/fafde5f25bc44227e7284b0ca24dd8a5e677543e fix this issue